packages/clerk-js/src/ui/components/SignUp/__tests__/SignUpVerifyEmail.test.tsx

```diff
@@ -19,8 +20,8 @@ describe('SignUpVerifyEmail', () => {
       f.startSignUpWithEmailAddress({ emailAddress: 'test@clerk.com' });
     });
     fixtures.signUp.prepareEmailAddressVerification.mockRejectedValue(null);
-    render(<SignUpVerifyEmail />, { wrapper });
-    screen.getByText('test@clerk.com');
+    const { findByText } = render(<SignUpVerifyEmail />, { wrapper });
+    await waitFor(async () => expect(await findByText('test@clerk.com')).toBeInTheDocument());
```

Comment:
❓ is the `waitFor` needed? Doesn't the `findBy*` use wait internally?

Reply:
It is needed, if i removed it the test passes, but we still get the "act" errors.